packages/teams-js/test/utils.ts

```diff
@@ -138,6 +138,20 @@ export class Utils {
     return null;
   };
 
+  public findKthMessagesByFunc = (func: string, _k: number): MessageRequest | null => {
```

Comment:
Can you please explain why we need this function and why can't we use the existing `findMessageByFunc`?

Reply:
In my case, there are two 'registerHandler' events fired in order at the same time, I can't get the second fired event use `findMessageByFunc`.